docs/reference/catalog-reference.md

```diff
@@ -1,6 +1,320 @@
+
+<h5> Example </h5>
+
+```python
+catalog.download_quicklooks(
+    image_ids=list(search_results.id),  # Use catalog.search() to get search_results
+    collection="phr",  # Use catalog.get_collections() to select a collection name
+    output_directory="/Users/max.mustermann/Desktop/",
+)
+```
+
+## Orders
+
+### construct_order_parameters()
+
+The `construct_order_parameters()` function allows you to fill out an order form for catalog imagery.
+
+```python
+construct_order_parameters(
+    data_product_id,
+    image_id,
+    aoi,
+    tags,
+)
+```
+
+The returned format is `dict`.
+
+<h5> Arguments </h5>
+
+| Argument          | Overview                                                                                            |
+| ----------------- | --------------------------------------------------------------------------------------------------- |
+| `data_product_id` | **str / required**<br/>The data product ID.                                                         |
+| `image_id`        | **str / required**<br/>The full scene ID.                                                           |
+| `aoi`             | **Union[dict, Feature, FeatureCollection, list, GeoDataFrame, Polygon]**<br/>The AOI to be ordered. |
+| `tags`            | **List[str]**<br/>A list of tags that categorize the order.                                         |
+
+<h5> Example </h5>
+
+```python
+catalog.construct_order_parameters(
+    data_product_id="647780db-5a06-4b61-b525-577a8b68bb54",  # Use catalog.get_data_products(basic=False) to select a data product ID
+    image_id="a4c9e729-1b62-43be-82e4-4e02c31963dd",  # Use catalog.search() to select a full scene ID
+    aoi={
+        "type": "Polygon",
+        "coordinates": (
+            (
+                (13.375966, 52.515068),
+                (13.375966, 52.516639),
+                (13.378314, 52.516639),
+                (13.378314, 52.515068),
+                (13.375966, 52.515068),
+            ),
+        ),
+    },
+    tags=["project-7", "optical"],
+)
+```
+
+### estimate_order()
+
+The `estimate_order()` function returns a cost estimation for a catalog order.
+
+```python
+estimate_order(order_parameters)
+```
+
+The returned format is `int`.
+
+<h5> Arguments </h5>
+
+| Argument           | Overview                                                                      |
+| ------------------ | ----------------------------------------------------------------------------- |
+| `order_parameters` | **Union[dict, None] / required**<br/>Parameters with which to place an order. |
+
+<h5> Example </h5>
+
+```python
+catalog.estimate_order(order_parameters) # Use catalog.construct_order_parameters() to get order_parameters
+```
+
+## Visualization
+
+### plot_coverage()
+
+The `plot_coverage()` function allows you to visualize the coverage of your search results.
+Use together with [`search()`](#search).
+
+```python
+plot_coverage(
+    scenes,
+    aoi,
+    legend_column,
+    figsize,
+)
+```
+
```

Comment:
@dustgalactic 
`plot_coverage()` and `plot_quicklooks()`  functions both return None so I've omitted the usual return type sentence here. Let me know if that makes sense.

> If your function performs actions but doesn’t have a clear and useful return value, then you can omit returning None because doing that would just be superfluous and confusing.

https://realpython.com/python-return-statement/#returning-none-explicitly

Reply:
Sure, I did the same for these types of functions.